blockchain/types/transaction_signing.go

```diff
@@ -435,3 +568,13 @@ func deriveChainId(v *big.Int) *big.Int {
 	v = new(big.Int).Sub(v, big.NewInt(35))
 	return v.Div(v, common.Big2)
 }
+
+func decodeSignature(sig []byte) (r, s, v *big.Int) {
+	if len(sig) != crypto.SignatureLength {
+		panic(fmt.Sprintf("wrong size for signature: got %d, want %d", len(sig), crypto.SignatureLength))
+	}
+	r = new(big.Int).SetBytes(sig[:32])
+	s = new(big.Int).SetBytes(sig[32:64])
+	v = new(big.Int).SetBytes([]byte{sig[64] + 27})
```

Comment:
In Klaytn, no signer add 27 for `v`. 
```suggestion
	v = new(big.Int).SetBytes([]byte{sig[64] + 35})
```